docs/standard/glossary.md

```diff
@@ -0,0 +1,228 @@
+---
+title: .NET Glossary
+description: Find out the meaning of selected terms used in the .NET documentation.
+keywords: .NET glossary, .NET dictionary, .NET terminology, .NET platform, .NET framework, .NET runtime
+author: tdykstra
+ms.author: tdykstra
+ms.date: 07/08/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# .NET Glossary
+
+The primary goal of this glossary is to clarify meanings of selected terms and acronyms that appear frequently in the .NET documentation without definitions. Some of these terms are used inconsistently, so a secondary goal is to help reduce the confusion that inconsistent usage causes.
+
+## AOT
+
+Ahead-of-time compiler. Similar to JIT, this compiler also translates IL to machine code. In contrast to JIT compilation, AOT compilation happens before the application is executed and is usually performed on a different machine. AOT tool chains don't trade runtime for compile time and thus can spend more time optimizing. Since the context of AOT is the entire application, the AOT compiler can also perform cross-module linking and whole-program analysis, which means that all references are followed and a single executable is produced.
```

Comment:
If there's a strong case for it, it'll prob be that with localization, they'll say that if it isn't a complete sentence that the auto-translation will flake out on a bunch and the translations won't be so hot. Just guessing tho from other comments I've seen regarding how they localize content.

The application to app thing was from the chats I've had mostly with [@}RickAnderson. I try to follow his advice as best I can.